frontend/components/Dashboard/PostCard/postCardUtils.ts

```diff
@@ -0,0 +1,43 @@
+import { format } from 'date-fns'
+import { sanitize } from '../../../utils'
+import { User, PostStatus, Thread } from '../../../generated/graphql'
+
+// TODO: this will be unnecessary once we use types from generated graphql
```

Comment:
Yep we're already using the generated GraphQL types. I just had trouble in the past with the `Post` type in particular, as it had been generated incorrectly somehow, but this may be fixed now since we've improved all the backend stuff a ton. It's worth giving it a try since it came up in this PR a few times

Reply:
I will give this a try again shortly! But I believe when I used the generated graphql type, the Post didn't contain several attributes that I needed (`excerpt`, `readTime`, `likes`, etc.)